### PR TITLE
Fixes some static analysis warnings and recommendations

### PR DESCRIPTION
**Pull request by yurydelendik**
- Useless conditional
- Superfluous trailing arguments
- Useless assignment to local variable
- Misspelled identifier
- JSDoc tag for non-existent parameter

### CODE REVIEW

File: src/core/fonts.js [hunk 1]
```diff
@@ -55,24 +54,18 @@ var isInt = sharedUtil.isInt;
...
 var shadow = sharedUtil.shadow;
-var stringToBytes = sharedUtil.stringToBytes;
 var string32 = sharedUtil.string32;
 var warn = sharedUtil.warn;
 var MissingDataException = sharedUtil.MissingDataException;
 var Stream = coreStream.Stream;
 var Lexer = coreParser.Lexer;
 var getGlyphsUnicode = coreGlyphList.getGlyphsUnicode;
 var getDingbatsGlyphsUnicode = coreGlyphList.getDingbatsGlyphsUnicode;
-var ISOAdobeCharset = coreCharsets.ISOAdobeCharset;
```
Comment (timvandermeij) [inline src/core/fonts.js]: We can probably remove the `coreCharsets` import now that these lines are gone. `cff_parser` imports it already for us.

Comment (yurydelendik) [inline src/core/fonts.js]: You are right. Thanks.

**Comment by timvandermeij** [comment]
Awesome work!

**Pull request merged**